app/src/main/java/org/ole/planet/myplanet/base/BaseExamFragment.kt

```
@@ -204,8 +204,11 @@ abstract class BaseExamFragment : Fragment(), ImageCaptureCallback {
             etAnswer.addTextChangedListener(answerTextWatcher)
         } else {
             answerTextWatcher = object : TextWatcher {
+                @Suppress("EmptyMethod")
                 override fun beforeTextChanged(charSequence: CharSequence, i: Int, i1: Int, i2: Int) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)